pay-api/src/pay_api/resources/v1/eft_short_names.py

```diff
@@ -0,0 +1,112 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Resource for Payment account."""
+from http import HTTPStatus
+
+from flask import Blueprint, current_app, jsonify, request
+from flask_cors import cross_origin
+
+from pay_api.exceptions import BusinessException, error_to_response
+from pay_api.schemas import utils as schema_utils
+from pay_api.services.eft_short_names import EFTShortnames as EFTShortnameService
+from pay_api.utils.auth import jwt as _jwt
+from pay_api.utils.endpoints_enums import EndpointEnum
+from pay_api.utils.enums import Role
+from pay_api.utils.errors import Error
+from pay_api.utils.trace import tracing as _tracing
+
+bp = Blueprint('EFT_SHORT_NAMES', __name__, url_prefix=f'{EndpointEnum.API_V1.value}/eft-shortnames')
+
+
+@bp.route('', methods=['GET', 'OPTIONS'])
+@cross_origin(origins='*', methods=['GET', 'POST'])
+@_jwt.requires_auth
+@_jwt.has_one_of_roles([Role.SYSTEM.value, Role.STAFF.value])
+def get_eft_shortnames():
+    """Get all eft short name records."""
+    current_app.logger.info('<get_eft_shortnames')
+
+    include_all: bool = bool(request.args.get('includeAll', 'false').lower() == 'true')
+    page: int = int(request.args.get('page', '1'))
+    limit: int = int(request.args.get('limit', '10'))
+
+    response, status = EFTShortnameService.search(include_all, page, limit), HTTPStatus.OK
+    current_app.logger.debug('>get_eft_shortnames')
+    return jsonify(response), status
+
+
+@bp.route('', methods=['POST', 'OPTIONS'])
+@cross_origin(origins='*')
+@_jwt.requires_auth
+@_jwt.has_one_of_roles([Role.SYSTEM.value, Role.STAFF.value])
+def post_eft_shortname():
```

Comment:
not used, dont' think we need this route, as discussed - hopefully we don't

Reply:
Removed POST route